### ISSUE TITLE
Accessibility Quiz Step 16

### ISSUE THREAD

**Issue by jeremylt** [freeCodeCamp/freeCodeCamp#49243]
### Describe the Issue

The wording here is awkward: 

> To the `section` elements, give the following `aria-labelledby` attributes:

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-accessibility-by-building-a-quiz/step-16

### Your code

N/A

### Expected behavior

Simpler grammatical structure should be used in the curriculum:

> Add the following `aria-labelledby` attributes to the `section` elements:

### System

All

**Comment by bbsmooth** [comment]
Hate to hijack this issue, but while we are rewording this step, the first sentence needs some help as well. Ya, I know I'm being very picky here. We accessibility nerds are like that.

> Every region role requires a visible label, which should be referenced by the aria-labelledby attribute.

Regions do not require a visible label. What they do require is an accessible name, which can be given by either `aria-labelledby` or `aria-label`. If there is an element on the page that can be used for the accessible name (such as a heading element inside the region) then definitely use `aria-labelledby` for the accessible name. If there is no appropriate element on the page then use `aria-label`.

My first shot at a rewrite.

> Every `region` role requires a label, which helps screen reader users understand the purpose of the region. One method for adding a label is to add a heading element inside the region and then reference it with the `aria-labelledby` attribute.

Also, I think it might make more sense to rearrange the steps in the instructions to have them add the headings with `id`s first and then the `aria-labelledby` attributes on the `div`s. But I can live with the way it is now.

**Comment by jeremylt** [comment]
Sounds good to me.

Then, this issue is to change

> Every `region` role requires a visible label, which should be referenced by the `aria-labelledby` attribute.

> To the `section` elements, give the following `aria-labelledby` attributes:

into 

> Every `region` role requires a label, which helps screen reader users understand the purpose of the region. One method for adding a label is to add a heading element inside the region and then reference it with the `aria-labelledby` attribute.

> Add the following `aria-labelledby` attributes to the `section` elements:

**Comment by Sriparno08** [comment]
Alright, I am going ahead!
